.github/workflows/go-test.yaml

```diff
@@ -153,7 +153,7 @@ jobs:
           set -ex
           kubectl rook-ceph destroy-cluster
           sleep 1
-          kubectl get deployments -n rook-ceph --no-headers| wc -l | (read n && [ $n -le 1 ] || { echo "the crs could not be deleted"; exit 1;})
+          kubectl get deployments -n rook-ceph --no-headers| wc -l | (read n && [ $n -le 1 ] || { echo "the crs could not be deleted"; kubectl get all -n rook-ceph; exit 1;})
```

Comment:
why we cant simply use `kubectl wait`

Reply:
we don't need to wait as the wait time is in the main go code in last line we are just checking if any ceph cluster cr is not deleted

Reply:
got it